pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -99,6 +99,7 @@ ValidationType:
     - REQUIRED
     - UNIQUE
     - YEAR_REQUIRED
+    - UNIQUE_VALUE
```

Comment:
Considered validation rule name STUDY_UNIQUE_VALUE so that scope of the uniqueness will be evident in the name but felt like adding notes in the code and API Doc might be good enough.